src/infrastructure/jira/schemas.ts

```diff
@@ -16,3 +23,41 @@ export const ATTACHED_DESIGN_URL_V2_VALUE_SCHEMA: JSONSchemaTypeWithId<
 		required: ['url', 'name'],
 	},
 };
+
+export const NODE_DEV_STATUS_SCHEMA: JSONSchemaType<NodeDevStatus> = {
```

Comment:
**nitpick:** @stephbrook11 , consider exporting only top-level schemas (e.g., `FILE_RESPONSE_SCHEMA`) and keeping other schemas private, as for example, `NODE_DEV_STATUS_SCHEMA`.